modules/ml/pom.xml

```diff
@@ -102,7 +102,6 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-context</artifactId>
             <version>${spring.version}</version>
-            <scope>test</scope>
```

Comment:
We really need this change?

Reply:
yep, we use spring in the resolving resource path